athom-smart-plug.yaml

```diff
@@ -311,6 +313,7 @@ switch:
     id: relay
     restore_mode: ${relay_restore_mode}
     icon: mdi:${power_plug_type}
+    internal: "{$hide_switch}"
```

Comment:
Shouldn't this be `"${hide_switch}"` like the rest of the substitutions? Does this string-interpolation method work for you?